ssl/statem/statem_srvr.c

```diff
@@ -161,6 +168,11 @@ static int ossl_statem_server13_read_transition(SSL_CONNECTION *s, int mt)
             st->hand_state = TLS_ST_SR_KEY_UPDATE;
             return 1;
         }
+
+        if (mt == DTLS13_MT_ACK) { /*TODO(DTLSv1.3): Delete?*/
```

Comment:
Presumably we could get spurious ACKs at any time? Should we just ignore them?

Reply:
I think it is best to ignore them because we only require them in certain states.